modules/network/dhcp-1/python/src/grpc_server/dhcp_leases.py

```diff
@@ -56,20 +56,21 @@ def get_lease(self, hw_addr):
 
   def get_leases(self):
     leases = []
-    lease_list_raw = self._get_lease_list()
-    LOGGER.info('Raw Leases:\n' + str(lease_list_raw) + '\n')
-    lease_list_start = lease_list_raw.find('=========', 0)
-    lease_list_start = lease_list_raw.find('\n', lease_list_start)
-    lease_list = lease_list_raw[lease_list_start + 1:]
+    lease_list = self._get_lease_list()
+    LOGGER.info('Raw leases:\n' + str(lease_list) + '\n')
     lines = lease_list.split('\n')
     for line in lines:
+
+      # Ignore non-lease lines
+      if line.contains('Reading leases from') or line.contains('==='):
+        continue
+
       try:
         lease = DHCPLease(line)
         leases.append(lease)

```

Comment:
This change appears to break lease resolving and all DHCP tests, needs to be corrected or reverted.